### PR TITLE
feat: create, get, update, delete, get single

### PR DESCRIPTION
**Pull request by sunbryely-influxdata**
List Orgs
Create Org
Update Org
Get Org
Delete Org

### CODE REVIEW

File: src/cloud/tags.yml [hunk 1]
```diff
@@ -42,6 +42,13 @@ tags:
   - name: Delete
     description: |
       Delete data from an InfluxDB bucket.
+  - name: Organizations
+    description: |
+      Manage your [organization]({{% INFLUXDB_DOCS_URL %}}/reference/glossary/#organization).
+      An organization is a workspace for a group of users. Organizations can be
+      used to separate different environments within InfluxDB.
```
Comment (jstirnaman) [inline src/cloud/tags.yml:49]: Maybe clarify what is meant by "different environments"?

File: src/cloud/tags.yml [hunk 2]
```diff
@@ -42,6 +42,13 @@ tags:
...
+
+      Use the `/api/v2/orgs` endpoints to view, and manage organizations.
```
Comment (jstirnaman) [inline src/cloud/tags.yml:51]:
```suggestion
      Use the `/api/v2/orgs` endpoints to view and manage organizations.
```

File: src/common/responses/example-responses/PostOrgsSuccessResponse.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+  successResponse:
```
Comment (jstirnaman) [inline src/common/responses/example-responses/PostOrgsSuccessResponse.yml:1]: Why store these in separate files? Do you expect to reuse them elsewhere?

File: src/common/paths/orgs.yml [hunk 4]
```diff
@@ -13,24 +19,38 @@ get:
...
-      description: Filter organizations to a specific organization ID.
+      description: |
+        The ID of the organization to retrieve.
     - in: query
       name: userID
       schema:
         type: string
-      description: Filter organizations to a specific user ID.
+      description: |
+        The ID of the user to filter with.
```
Comment (jstirnaman) [inline src/common/paths/orgs.yml:35]:
```suggestion
        A user ID.
```

File: src/common/paths/orgs.yml [hunk 5]
```diff
@@ -42,6 +62,17 @@ post:
   tags:
     - Organizations
   summary: Create an organization
+  description: |
+    Creates an [organizations]({{% INFLUXDB_DOCS_URL %}}/reference/glossary/#organization).
```
Comment (jstirnaman) [inline src/common/paths/orgs.yml:66]:
```suggestion
    Creates an [organization]({{% INFLUXDB_DOCS_URL %}}/reference/glossary/#organization).
```

File: src/common/paths/orgs.yml [hunk 6]
```diff
@@ -42,6 +62,17 @@ post:
...
+
+    #### InfluxDB Cloud
+
+    - You can't create additional organizations in InfluxDB Cloud. Only the
+    default organization is available.
```
Comment (jstirnaman) [inline src/common/paths/orgs.yml:71]:
```suggestion
    - Doesn't allow creating additional organizations with the API; only the
    default organization is available.
```

File: src/common/paths/orgs_orgID.yml [hunk 7]
```diff
@@ -30,6 +61,27 @@ patch:
...
     - Organizations
   summary: Update an organization
+  description: |
+    Updates an organization.
+
+    Use this endpoint to update properties
+    (`name`, `description`) of an organization.
+
+    Updating an organization’s name will affect any assets that reference the
+    organization by name, including the following:
```
Comment (jstirnaman) [inline src/common/paths/orgs_orgID.yml:71]:
```suggestion
    Updating an organization’s name affects all resources that reference the
    organization by name, including the following:
```

File: src/common/paths/orgs_orgID.yml [hunk 8]
```diff
@@ -30,6 +61,27 @@ patch:
...
+
+    - Queries
+    - Dashboards
+    - Tasks
+    - Telegraf configurations
+    - Templates
+
+    If you change an organization name, be sure to update the organization in
+    the above places as well.
```
Comment (jstirnaman) [inline src/common/paths/orgs_orgID.yml:80]:
```suggestion
    If you change an organization name, be sure to update the organization name in
    these resources as well.
```

File: src/common/paths/orgs_orgID.yml [hunk 9]
```diff
@@ -30,6 +61,27 @@ patch:
...
+
+    #### Related Guides
+
+    - [Update an organization]({{% INFLUXDB_DOCS_URL %}}/organizations/update-org/)
   requestBody:
     description: Organization update to apply
```
Comment (jstirnaman) [inline src/common/paths/orgs_orgID.yml:86]:
```suggestion
    description: The organization update to apply.
```

File: src/common/paths/orgs_orgID.yml [hunk 10]
```diff
@@ -44,14 +96,25 @@ patch:
...
       required: true
-      description: The ID of the organization to get.
+      description: |
+        The ID of the organization to update.
   responses:
     "200":
-      description: Organization updated
+      description:
+        Success.
+        The response body contains the update organization information.
```
Comment (jstirnaman) [inline src/common/paths/orgs_orgID.yml:105]:
```suggestion
        The response body contains the updated organization.
```

File: src/common/paths/orgs_orgID.yml [hunk 11]
```diff
@@ -63,23 +126,73 @@ delete:
   tags:
     - Organizations
   summary: Delete an organization
+  description: |
+    Deletes an organization.
+
+    Deleting an organization from InfluxDB Cloud cannot be undone.
```
Comment (jstirnaman) [inline src/common/paths/orgs_orgID.yml:132]:
```suggestion
    Deleting an organization from InfluxDB Cloud can't be undone.
```

File: src/common/paths/orgs_orgID.yml [hunk 12]
```diff
@@ -63,23 +126,73 @@ delete:
...
+          - The organization is deleted.
+    "400":
+      $ref: "../responses/BadRequestError.yml"
+    "401":
+      $ref: "../responses/AuthorizationError.yml"
     "404":
-      description: Organization not found
+      description: |
+        Not found.
+        Organization not found.
```
Comment (jstirnaman) [inline src/common/paths/orgs_orgID.yml:181]:
```suggestion
        InfluxDB can't find the organization.
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]

File: src/common/responses/example-responses/PostOrgsSuccessResponse.yml (hunk 3 above)
Comment (sunbryely-influxdata) [reply src/common/responses/example-responses/PostOrgsSuccessResponse.yml:1]: Some of the other responses are extremely large and I wanted to separate those into different files to keep the file cleaner.

**Review by jstirnaman — APPROVED** [review]
Resolve conflicts.

**Pull request merged**